xml/admin_nfsganesha.xml

```diff
@@ -556,6 +557,7 @@ EXPORT {
     default, client libraries do not log errors or any other output. To see
     more details about &ganesha; interacting with the &ceph; cluster (for
     example, connection issues details) logging needs to be explicitly defined
+<!-- 2020-09-7 tbazant (ceph.conf): is the following still valid and usable? -->
```

Comment:
Have pinged @bk201 and @supriti regarding this line. If not, @mgfritch could you please take a look?

Reply:
`/etc/ceph/ceph.con` becomes a file in the container which is bind-mounted from host (e.g. `/var/lib/ceph/ae0e6b0b-fbd6-49ae-883b-3ed03011ece8/nfs.bar.mgr0/config`). Need a reliable way to locate this file.